packages/fiori/src/themes/ShellBar.css

```diff
@@ -35,6 +35,11 @@
 	font-weight: bold;
 }
 
+.ui5-shellbar-menu-button,
```

Comment:
IE fails when selectors such as :host, ::slotted are mixed with other selectors.
Same with the IE CSS processing script. 
On IE the button focus looks broken (black) maybe this is part of the reason?